modules/troll.js

```diff
@@ -12,4 +12,9 @@ module.exports.tablethrow = function(message){
 
 module.exports.lenny = function(message){
   message.channel.send("( ͡° ͜ʖ ͡°)")
-}
\ No newline at end of file
+};
+
+module.exports.shame = function(message){
+	message.channel.send("SHAME SHAME\nhttps://www.youtube.com/watch?v=SrDSqODtEFM")
```

Comment:
Spaces are superior to tabs, but worst of all is having them mixed. Let's not mix them; it fucks with text editors. You can set your text editor to replace tabs with spaces, which is what I personally do.